src/System.CommandLine/Help/HelpBuilder.cs

```diff
@@ -639,9 +636,18 @@ string GetArgumentDescription()
                     return $"{name}: {ownDefaultValue}";
                 }
             }
+            return $"{name}: {GetArgumentDefaultValue(argument)}";
+        }
 
+        /// <summary>
+        /// Gets the argument default value to be displayed for help purposes.
+        /// </summary>
+        /// <param name="argument">The argument to get the description for.</param>
+        public string GetArgumentDefaultValue(
```

Comment:
```suggestion
        public string GetArgumentDefaultValueDescription(
```

This might be a useful rename since it's not actually returning the default value.

Reply:
Here description of parameter was actually wrong, fixed it. The method actually returns default value, not the descriptions.